test/app/dftb+/bin/globalpostrun.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+#------------------------------------------------------------------------------#
+#  DFTB+: general package for performing fast atomistic simulations            #
+#  Copyright (C) 2006 - 2021  DFTB+ developers group                           #
+#                                                                              #
+#  See the LICENSE file for terms of usage and distribution.                   #
+#------------------------------------------------------------------------------#
+
+############################################################################
+#
+#  Script for automated testing outside of tagged files
+#
+############################################################################
+
+if grep -l NaN output; then
```

Comment:
Check for whole words here?

Reply:
How about 
grep -l "\<NaN\>"
?

Reply:
Unfortunately only on linux, not BSD derived (my construct might also fail there): https://stackoverflow.com/questions/9792702/does-bash-support-word-boundary-regular-expressions

Reply:
How many DFTB+ users do we have on BSD anyway? cc @aradi ;)

Reply:
I think, we should probably still keep compatibility, if possible, in order to avoid bug reports from BSD-packagers. :wink: